apps/public-reference/src/customScripts.ts

```diff
@@ -0,0 +1,34 @@
+export const headScript = () => {
+  const gtmKey = process.env.gtmKey
```

Comment:
Do we want to have the process env key be all caps to be consistent with the other ones in .env?

Reply:
The env keys within the app are different than the actual server-side env keys. There's a mapping from node to SRR/client-side that happens in next.config.js — so you'll want your actual env var set via `GTM_KEY`. Let me know if that works.